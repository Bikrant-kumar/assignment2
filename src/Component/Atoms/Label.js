import React from 'react'
import Box from '@material-ui/core/Box';

function Label(props) {
    const {
        text
       } = props;
    return (
        <Box>
            {text}
        </Box>
    )
}

export default Label