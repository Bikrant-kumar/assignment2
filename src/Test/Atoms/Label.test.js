import React from 'react';
import { screen,render } from '@testing-library/react';
import Label from '../../Component/Atoms/Label'
import renderer from 'react-test-renderer';

describe(Label ,() => {

   test('Label check',() => {
      render(<Label text='Name' />);
     expect(screen.getByText("Name")).toBeInTheDocument();
  });

   test('test Label with snapshot',()=>{
      const component= renderer.create(<Label text='Name'/>).toJSON();
      expect(component).toMatchSnapshot();
   });

})