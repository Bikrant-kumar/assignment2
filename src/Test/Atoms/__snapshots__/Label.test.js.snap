// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label test Label with snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-2"
>
  Name
</div>
`;
